tests/config/nextflow.config

```
@@ -28,6 +30,9 @@ if ("$PROFILE" == "singularity") {
     docker.userEmulation = false
     docker.fixOwnership = true
     docker.runOptions = '--platform=linux/amd64 -u $(id -u):$(id -g)'
+    wave.enabled = true
+    wave.strategy = ['dockerfile', 'conda', 'container']
```

```suggestion
    wave.strategy = 'conda'
```

Likely you want only rely on `conda` packages

---

We have a few modules with dockerfiles (bclconvert off the top of my head) and a few that use images only and don't have conda packages (parabricks I think is this way)